tests/apps/courses/test_admin_course_run.py

```diff
@@ -209,12 +208,12 @@ def test_admin_course_run_change_view_get_superuser_draft(self):
         # Check that the page includes all our fields
         self.assertContains(response, "id_title", count=2)
         self.assertContains(response, "id_resource_link", count=2)
-        self.assertContains(response, "id_start_", count=3)
-        self.assertContains(response, "id_end_", count=3)
-        self.assertContains(response, "id_enrollment_start_", count=3)
-        self.assertContains(response, "id_enrollment_end_", count=3)
-        self.assertContains(response, "id_languages", count=2)
-        self.assertContains(response, "id_enrollment_count", count=2)
+        self.assertContains(response, "id_start_", count=2)
+        self.assertContains(response, "id_end_", count=2)
+        self.assertContains(response, "id_enrollment_start_", count=2)
+        self.assertContains(response, "id_enrollment_end_", count=2)
+        self.assertContains(response, "id_languages", count=3)
+        self.assertContains(response, "id_enrollment_count", count=3)
```

Comment:
Why all the count are changing ?

Reply:
When I set the version 4.1 of DJango, this test failed by itself. It seems like django might have change their template frendering for admin views. By counting in the response.content, we can find that change. I will count them again to confirm you the count found between version 3.2, 4.0, and 4.1.

Reply:
After my investigation, once we pass to the versionn 4.1 of Django, this test suddenly fails and needs some adjustments. 
The new count is accurate.  Plus, I've checked the creation on the django admin side of Richie and it works to create course runs in each versions (3.2, 4.0, 4.1, and 4.2.13). All the fields in the forms work and my resource is created at the end of the form. There are no errors and the page is accessible.

Reply:
Using the DOM with the inspector:

When the count decreases by 1: In version 4.1, the label ("Début de cours" - Course start) no longer has the string of the type id_start_, totaling 2 occurrences, for example, for the 2 fields to fill containing the searched string. When we are below version 4.1, we have the label and the 2 fields that have the searched string, totaling 3 occurrences.

When the count increases by 1: In version 4.1, a new tag is added to the template for certain fields of the type chainerecherché_helptext that we do not have in version < 4.1 (for example: id_languages_helptext or id_enrollment_count_helptext). In versions <4.1, the string was simply "help".